sql-indent.el

```diff
@@ -1752,7 +1755,10 @@ This is an indentation adjuster and needs to be added to the
     ;; consider them an operator and line up the line to the first word of the
     ;; line, not the operator
     (cond ((looking-at "\\W+")
-	   (let ((ofs (length (match-string 0))))
+	   (let* ((operator (match-string 0))
+		  (ofs (length operator)))
+	     (when (string-match "'$" operator)
+	       (setq ofs (1- ofs)))
```

Comment:
I think the `(forward-sexp -1)` line below is the one that needs to change.  Perhaps we will need to write our own "forward/backward-term" function.   I think all other places were `forward-sexp` is used will have the same problem.  I will think about how to best do this.